packages/lodestar/src/network/encoders/utils.ts

```diff
@@ -46,3 +46,10 @@ export function getDecompressor(encoding: ReqRespEncoding): IDecompressor {
 export function encodeResponseStatus(status: number): Buffer {
   return Buffer.from([status]);
 }
+
+export function maxEncodedLen(sszLength: number, encoding: ReqRespEncoding): number {
+  if (encoding === ReqRespEncoding.SSZ) {
+    return sszLength;
+  }
+  return 32 + sszLength + sszLength / 6;
```

Comment:
Is this padding from snappy compression? would you mind adding a brief comment on what/why this is?

Reply:
yeah added that. In the spec of p2p we have the `maxEncodedLen()` function too so this is just a 1-1 mapping. This is the worst-case compression for ssz and ssz-snappy.